spec.md

```diff
@@ -152,13 +152,8 @@ of a JSON serialization, the context attributes and the event data might
 both appear within the same JSON object.
 
 ### Extension Attributes
-CloudEvent producers MAY include additional extension attributes within the
-event. This enables event producers, or middleware, to include additional
-metadata that might be used for any purpose in the processing of the
-CloudEvent, such as identifying or correlating event sources. See
-[CloudEvent Attributes Extensions](primer.md#cloudevent-attribute-extensions)
```

Comment:
I think the last sentence, including this link, should remain.

Reply:
Will add it back to the extension section.